lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -423,11 +454,10 @@ def vote_for_comment(request, course_id, comment_id, value):
     """
     given a course_id and comment_id,
     """
-    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
-    user = cc.User.from_django_user(request.user)
     comment = cc.Comment.find(comment_id)
-    user.vote(comment, value)
-    return JsonResponse(prepare_content(comment.to_dict(), course_key))
+    result = _vote_or_unvote(request, course_id, comment, value)
+    comment_voted.send(sender=None, user=request.user, post=comment)
```

Comment:
This line should be removed.  `comment_voted` doesn't exist in this version of platform and was added later.